src/main/scala/com.snowplowanalytics.snowplow.analytics.scalasdk/Event.scala

```diff
@@ -256,4 +256,18 @@ object Event {
     */
   def parse(line: String): DecodeResult[Event] =
     parser.parse(line)
+
+  /**
+    * Creates empty event which optional fields of it are None
+    * Only necessary fields are given as arguments
+    */
+  def emptyEvent(id: UUID, collectorTstamp: Instant, vCollector: String, vTstamp: String): Event =
```

Comment:
What is `vTstamp`?

Reply:
It is v_etl. I think it is better to change its name to vEtl.